### PR TITLE
Fix cloud service doc

### PR DESCRIPTION
**Pull request by szihai**
Added missing vela.yaml. Changed site_bar as well.

### CODE REVIEW

File: docs/en/platform-engineers/cloud-services.md [hunk 1]
```diff
@@ -2,87 +2,130 @@
...
-  * Add cloud provider: `kubectl crossplane install provider crossplane/provider-alibaba:v0.3.0`
-  * Create provider secret: `kubectl create secret generic alibaba-creds --from-literal=accessKeyId=<change here> --from-literal=accessKeySecret=<change here> -n crossplane-system`
-  * Configure the provider: `kubectl apply -f script/provider.yaml`
-* Configure infrastructure: `kubectl crossplane install configuration crossplane/getting-started-with-alibaba:v0.13`
-
-So far we have configured Crossplane on the cluster.
-
-</details>
+In this tutorial, we use Crossplane as the cloud resource operator for Kubernetes.   
+Please follow the Crossplane [Documentation](https://crossplane.io/docs/) under the `Welcome` section to configure Crossplane with your cloud account.
```
Comment (resouer) [inline docs/en/platform-engineers/cloud-services.md:8]: please specify which version of crossplane you used to test the tutorial.

File: docs/en/platform-engineers/cloud-services.md [hunk 2]
```diff
@@ -2,87 +2,130 @@
...
-* Configure cloud provider(Alibaba Cloud) 
-  * Add cloud provider: `kubectl crossplane install provider crossplane/provider-alibaba:v0.3.0`
-  * Create provider secret: `kubectl create secret generic alibaba-creds --from-literal=accessKeyId=<change here> --from-literal=accessKeySecret=<change here> -n crossplane-system`
-  * Configure the provider: `kubectl apply -f script/provider.yaml`
-* Configure infrastructure: `kubectl crossplane install configuration crossplane/getting-started-with-alibaba:v0.13`
-
-So far we have configured Crossplane on the cluster.
-
-</details>
+In this tutorial, we use Crossplane as the cloud resource operator for Kubernetes.
```
Comment (resouer) [inline docs/en/platform-engineers/cloud-services.md:7]: Please use preview to format the doc a bit.

Caps, paragraphs, etc.

File: docs/en/platform-engineers/cloud-services.md [hunk 3]
```diff
@@ -2,87 +2,130 @@
...
-  * Create provider secret: `kubectl create secret generic alibaba-creds --from-literal=accessKeyId=<change here> --from-literal=accessKeySecret=<change here> -n crossplane-system`
-  * Configure the provider: `kubectl apply -f script/provider.yaml`
-* Configure infrastructure: `kubectl crossplane install configuration crossplane/getting-started-with-alibaba:v0.13`
-
-So far we have configured Crossplane on the cluster.
-
-</details>
+In this tutorial, we use Crossplane as the cloud resource operator for Kubernetes.   
+Please follow the Crossplane [Documentation](https://crossplane.io/docs/) under the `Welcome` section to configure Crossplane with your cloud account.   
+**Note: When installing crossplane helm chart, please do not include the `--set alpha.oam.enabled=true` flag as KubeVela has already installed these OAM crds.**
```
Comment (resouer) [inline docs/en/platform-engineers/cloud-services.md:9]: > Note: when installing Crossplane Helm chart, please don't set `alpha.oam.enabled=true` as OAM crds are already installed by KubeVela.

**Review by hongchaodeng — APPROVED** [review]
nice work!

lgtm

**Comment by hongchaodeng** [comment]
need rebase

File: docs/en/platform-engineers/cloud-services.md [hunk 4]
```diff
@@ -2,15 +2,16 @@
...
+
+**Note: When installing crossplane helm chart, please don't set `alpha.oam.enabled=true` as OAM crds are already installed by KubeVela.**
 
 ## Step 2: Add Workload Definition
 
-First,register the `rds` workload type to KubeVela:
+First, register the `rds` workload type to KubeVela.
 
 ```bash
 $ cat << EOF | kubectl apply -f -
```
Comment (zzxwill) [inline docs/en/platform-engineers/cloud-services.md:17]: This is cool which provides an example for applying an workload/traits besides built-in and `vela cap`.

**Pull request merged**